core/interaction_type.cpp

```diff
@@ -519,22 +522,38 @@ void InteractionType::CalculateAllDistances(Subpopulation *p_subpop)
 			
 			if (exerter_sex_ == IndividualSex::kUnspecified)
 			{
+
 				// Without a specified exerter sex, we can add each exerter with no sex test
+                //double start_time = omp_get_wtime();
+
 				switch (spatiality_)
 				{
 					case 1:
 						for (row = start_row; row < after_end_row; row++)
+                        {
 							BuildSA_1(subpop_data.kd_root_, position_data + row * SLIM_MAX_DIMENSIONALITY, row, subpop_data.dist_str_);
+                        }
 						break;
 					case 2:
-						for (row = start_row; row < after_end_row; row++)
+                    //ANNOTATE_SITE_BEGIN(s7)
+#ifdef Parallel
```

Comment:
I thought OMP's design was such that these lines did not need to be #ifdef'd, because when not doing a parallel build the omp.h header had stubs in place that rendered all of the parallel constructs no-ops.  No?